src/Engine/ProtoScript/Runners/LiveRunner.cs

```diff
@@ -1940,6 +1940,10 @@ public List<LibraryMirror> ResetVMAndImportLibrary(List<string> libraries)
         /// <returns></returns>
         public IDictionary<Guid, List<ProtoCore.Runtime.WarningEntry>> GetRuntimeWarnings()
         {
+            var ret = new Dictionary<Guid, List<ProtoCore.Runtime.WarningEntry>>();
+            if (runtimeCore == null)
```

Comment:
Thanks for this. Was there a simple workflow to get this bug? Im just curious to know when this is called when the rutnimeCore was null